Python_Engine/Compute/BasePythonEnvironment.cs

```diff
@@ -41,6 +41,10 @@ public static partial class Compute
             bool reload = true
         )
         {
+            if (!Directory.Exists(Query.DirectoryEnvironments()))
+                // create PythonEnvironments directory if it doesnt already exist
+                Directory.CreateDirectory(Query.DirectoryEnvironments());
```

Comment:
```suggestion
            if (!Directory.Exists(Query.DirectoryEnvironments()))
            {
                // create PythonEnvironments directory if it doesnt already exist
                Directory.CreateDirectory(Query.DirectoryEnvironments());
            }
```

While comments don't count for code - keeping them grouped and tidy does 😉